components/monitoring/prometheus/base/servicemonitors/dora-service.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: dora-service
+  namespace: appstudio-workload-monitoring
+  labels:
+    control-plane: controller-manager
+    prometheus: appstudio-workload
+    app: exporter
+spec:
+  endpoints:
+  - interval: 120s
+    path: /metrics
+    port: metrics-port
+    tlsConfig:
+      insecureSkipVerify: true
+    scrapeTimeout: 30s
+  namespaceSelector:
+    matchNames:
+    - dora-metrics
+  selector:
+    matchLabels:
+      app: exporter
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus-dora-metrics-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: dora-metrics-reader
```

Comment:
where is this `dora-metrics-reader` clusterRole defined?

Reply:
@bamachrn I was waiting on the same answer.

Reply:
This is from the respective repo. I looked in the dora-metrics repo 
https://github.com/redhat-appstudio/dora-metrics/blob/main/deploy/base/clusterrole.yaml
https://github.com/redhat-appstudio/dora-metrics/blob/main/deploy/base/clusterrolebinding.yaml

Reply:
@amfred @Michkov ^

Reply:
Right, thanks